mrtt-ui/src/components/ButtonDeleteForm.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { styled } from '@mui/system'
+import language from '../language'
+import themeMui from '../styles/themeMui'
+import ButtonSecondaryWithLoader from './ButtonSecondaryWithLoader'
+
+const StyledButton = styled(ButtonSecondaryWithLoader)`
```

Comment:
I updated this to match the delete button from the Landscape form which _I think_ instructed me on. Anyway, consistency at least. Its not red, its teal like everything else.